servlet/6.0/_index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Jakarta Servlet 6.0 (under development)"
+date: 2021-10-15
+summary: "Release for Jakarta EE 10"
+---
+Jakarta Servlet defines a server-side API for handling HTTP requests and responses.
+
+* [Jakarta Servlet 6.0 Release Record](https://projects.eclipse.org/projects/ee4j.servlet/releases/6.0)
+  * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
+* [Jakarta Servlet 6.0 Specification Document](./jakarta-servlet-spec-6.0.pdf) (PDF)
+* [Jakarta Servlet 6.0 Specification Document](./jakarta-servlet-spec-6.0.html) (HTML)
+* [Jakarta Servlet 6.0 Javadoc](./apidocs)
+* [Jakarta Servlet 6.0 TCK](https://download.eclipse.org/jakartaee/servlet/6.0/jakarta-servlet-tck-6.0.0.zip)([sig](https://download.eclipse.org/jakartaee/servlet/6.0/jakarta-servlet-tck-6.0.0.zip.sig),[sha](https://download.eclipse.org/jakartaee/servlet/6.0/jakarta-servlet-tck-6.0.0.zip.sha256),[pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
+* Maven coordinates
+  * [jakarta.servlet:jakarta.servlet-api:jar:6.0.0](https://search.maven.org/artifact/jakarta.servlet/jakarta.servlet-api/6.0.0/jar)
+
+# Plan
+
+The plan for 6.0.0 is:
+* remove currently deprecated features
+* consider removing / deprecating additional features
+* add clarity to existing features
+* implement requested enhancements
+
+Prioritisation to be determined by the community as work progresses.
+
+# Backwards compatibility
+Backward compatibility issue include:
```

Comment:
Thank you for explicitly explaining the reasons behind the major version update.